roles/container-engine/docker/molecule/default/molecule.yml

```diff
@@ -0,0 +1,37 @@
+---
+dependency:
+  name: galaxy
```

Comment:
we can disable galaxy i suppose as we are not using any roles from there, just add enabled: False

Reply:
Agreed. Thank you for pointing that out. You get Galaxy dependency by default upon "molecule init".

Reply:
I removed the dependency definition all together.